include/stdlib.h

```diff
@@ -216,6 +256,21 @@ FAR void *memalign(size_t, size_t);
 FAR void *zalloc(size_t);
 FAR void *calloc(size_t, size_t);
 
+#ifdef __cplusplus
+inline FAR void *aligned_alloc(size_t a, size_t s)
+{
+  return memalign(a, s);
```

Comment:
Is it OK for errno to get set by calling this?

Reply:
memalign don't set errno too, so errno isn't an issue here.

Reply:
Ah I see that now.  Looks like it should be, but is not implemented.